projects/word-weaver/src/app/core/local-storage/local-storage.service.ts

```diff
@@ -28,7 +29,7 @@ function slugify(str) {
   return str;
 }
 
-const APP_PREFIX = "ww-" + slugify(environment.base);
+const APP_PREFIX = "ww-" + slugify(environment.base) + CONFIG_NAME;
```

Comment:
```suggestion
const APP_PREFIX = "ww-" + slugify(environment.base) + slugify(CONFIG_NAME);
```

Might be worth slugifying as well? On the one hand this could unintentially create conflicts, on the other hand I'm not sure if localStorage can handle arbitrary characters

Reply:
Yes, excellent point, we should.